micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/JettyConnectionMetrics.java

```diff
@@ -162,10 +161,16 @@ public void onClosed(Connection connection) {
         }
 
         if (sample != null) {
-            String serverOrClient = connection instanceof HttpConnection ? "server" : "client";
+            String type = "UNKNOWN";
+            if (connection.getClass().getName().contains("server")) {
```

Comment:
We may want to consider switching to make the qualified type name the value of the `type` tag, but that would be breaking for anyone that relies on the current values. Hence I thought it better to consider that in a future version. This change should keep the prior values in most common cases, I hope.